### PR TITLE
Refactor Help Command to display updated guides

### LINKED ISSUE #97: Refactor help command

### PR DESCRIPTION
**Pull request by weiliann**
The current help command uses hard-coded help messages for each command in the system.

This approach causes frequent inconsistencies when commands are added or modified, as the help messages are not synchronized.

Refactor the help command to use the MESSAGE_USAGE string from each command, ensuring that the help information remains consistent and up-to-date.

Minor adjustments are made to some Command classes to ensure uniform message formatting across the system. Also, minor visual improvements were made to the help window for better user experience.

closes #97

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 1]
```diff
@@ -9,7 +9,9 @@ public class HelpCommand extends Command {
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays a summary of all commands available in the system.\n"
+            + "Parameters: None"
```
Comment (shizy) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java]: Add in a newline `\n` after "Parameters: None"

File: src/main/java/seedu/address/logic/commands/ListCommand.java [hunk 2]
```diff
@@ -9,12 +9,13 @@
...
 public class ListCommand extends Command {
-
     public static final String COMMAND_WORD = "list";
     public static final String COMMAND_ALIAS = "l";
-
     public static final String MESSAGE_SUCCESS = "Listed all persons";
-
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists all contacts in the system"
+            + "Parameters: None"
```
Comment (shizy) [inline src/main/java/seedu/address/logic/commands/ListCommand.java]: Add in a newline \n after "Parameters: None"

File: src/main/java/seedu/address/logic/commands/ListCommand.java [hunk 3]
```diff
@@ -9,12 +9,13 @@
...
  */
 public class ListCommand extends Command {
-
     public static final String COMMAND_WORD = "list";
     public static final String COMMAND_ALIAS = "l";
-
     public static final String MESSAGE_SUCCESS = "Listed all persons";
-
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists all contacts in the system"
```
Comment (shizy) [inline src/main/java/seedu/address/logic/commands/ListCommand.java]: Add in a newline \n after ": Lists all contacts in the system"

**Review by shizy — CHANGES_REQUESTED** [review]
Overall LGTM, but a few newline characters need to be added

File: src/main/java/seedu/address/logic/commands/HelpCommand.java (hunk 1 above)
Comment (weiliann) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java]: Thanks for spotting

**Review by shizy — APPROVED** [review]
LGTM

**Review by yyihaoc — APPROVED** [review]
LGTM!

**Pull request merged**